### DISCUSSION TITLE
Are there resources (books, videos, guides) for a beginner?

### DISCUSSION THREAD

**Discussion by khuezy** [airbnb/visx#1564] [General]
Hi, I'm new to visualization (I touched D3 in the past) - what's a good resource to learn Visx? I tried searching for books and videos but haven't found one that has good fundamentals. The Visx doc examples are good but it doesn't explain the concepts.

Thanks in advance.

**Comment by rookbreezy** [comment]
It would be nice to have a guide on how all the components fit together. Right now, there is a brief description of each component and some examples. If you want anything custom, good luck!

What is the intended learning path? Right now its:
- Learn all about SVG
- Learn all about D3
- Study docs / examples
- Study Visx source code
- If you get stuck ask your Airbnb colleague :)
